docs/s3cmd-with-minio.md

```diff
@@ -1,105 +1,129 @@
 # S3cmd with Minio Server [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-`S3cmd` is a CLI client for managing data in AWS S3, Google Cloud Storage or any cloud storage service provider that uses the s3 protocol.  `S3cmd` is open source and is distributed under the GPLv2 license.
+`S3cmd` is an open source CLI client for managing data in AWS S3, Google Cloud Storage, or any cloud storage service provider that uses the **s3** protocol. `S3cmd` is distributed under the [GPLv2 License](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html).
 
-In this recipe we will learn how to configure and use S3cmd to manage data with Minio Server.
+This guide describes how to configure `S3cmd` and use it to manage data on Minio Server. These are the steps you will follow:
 
-## 1. Prerequisites
+[1. Install Minio Server](#installminioserver)<br/>
+[2. Install `S3cmd`](#installs3cmd)<br/>
+[3. Configure `S3cmd`](#configures3cmd)<br/>
+[4. Run `S3cmd` Commands](#runs3cmdcommands)
 
-Install Minio Server from [here](http://docs.minio.io/docs/minio-quickstart-guide).
+## <a name="installminioserver"></a>1. Install Minio Server
 
-## 2. Installation
+Install Minio Server using the instructions in the [Minio Quickstart Guide](http://docs.minio.io/docs/minio-quickstart-guide).
 
-Install `S3cmd` from <http://s3tools.org/s3cmd>.
+## <a name="installs3cmd"></a>2. Install `S3cmd`
 
-## 3. Configuration
+Install `S3cmd` using these instructions: <http://s3tools.org/s3cmd>.
 
-We will run `S3cmd` on <https://play.minio.io:9000>.
+## <a name="configures3cmd"></a>3. Configure `S3cmd`
 
-Access credentials shown in this example belong to <https://play.minio.io:9000>. These credentials are open to public. Feel free to use this service for testing and development. Replace with your own Minio keys in deployment.
+### 3.1 Generate a Configuration file
+`S3cmd` uses a configuration file called **.s3cfg** to access cloud storage. Use the following command to generate the initial version of **.s3cfg**:
 
-Edit the following fields in your s3cmd configuration file `~/.s3cfg`
+```sh
+ `./S3cmd --configure`
+```
+
+**Note:** accept the defaults when prompted.
```

Comment:
With the version I used, s3cmd actually prompted for the same things we are documenting for the config file, do we need to document for both?

Reply:
@eco-minio I clarified this in the note indicating configuration to take place in the next step.